src/librustc_driver/Cargo.toml

```diff
@@ -15,6 +15,7 @@ log = "0.4"
 env_logger = { version = "0.5", default-features = false }
 rustc-rayon = "0.1.1"
 scoped-tls = { version = "0.1.1", features = ["nightly"] }
+getopts = "=0.2.17"
```

Comment:
Why are these `=` restrictions?

Reply:
To keep the existing crate version, which is what is used by some other tool (not sure which). Shall I change it?